napari/_qt/widgets/_tests/test_qt_color_swatch.py

```diff
@@ -0,0 +1,41 @@
+import itertools
+
+from qtpy.QtWidgets import QWidget
+
+from napari._qt.widgets.qt_color_swatch import QColorSwatch, QColorSwatchEdit
+
+
+def test_succesfull_create_qcolorswatchedit(qtbot):
+    parents = [
+        None,
+        QWidget(),
+    ]
+    colors = [
+        None,
+        [1, 1, 1, 1],
+    ]
+    tooltips = [
+        None,
+        'This is a test',
+    ]
+
+    for parent, color, tooltip in itertools.product(parents, colors, tooltips):
+        QColorSwatchEdit(parent, initial_color=color, tooltip=tooltip)
```

Comment:
it will be nice if you use `pytest.mark.parametrize` instead of inner for loop.

you should  also add all created widgets to cleaning using `qtbot.add_widget(widget)`